templates/redis.conf.j2

```diff
@@ -7,8 +7,8 @@ supervised no
 daemonize yes
 pidfile "/var/run/redis/redis-server.pid"
 logfile "{{ redis_server_log_file_path }}"
-loglevel "{{ redis_log_level }}"
-dir /var/lib/redis/
+loglevel {{ redis_log_level }}
```

Comment:
@christianhueserhzdr I was not able to reproduce this issue when upgrading from `6.2.6` to `6.2.7`. 

I tried the following steps:
1. `git checkout main`
2. Ensure `redis_version` in `defaults/main.yml` is `6.2.6`
3. `pipenv run molecule converge`
4. Optional: `pipenv run molecule login` and view the redis-server config
5. Change `redis_version` in `defaults/main.yml` to `6.2.7`
6. `pipenv run molecule converge`
7. Check if config has changed

Reply:
Do we need this change then, @christianhueserhzdr?

Reply:
No, we don't need this PR any longer. Thanks, Tobias @tobiashuste, for the reminder. I'll close it now.